website/pages/en/cookbook/timeseries.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Subgraph Best Practice 5 - Simplify and Optimize with Timeseries and Aggregations
+---
+
+## TLDR
+
+Leveraging the new timeseries and aggregations feature in subgraphs can significantly enhance both indexing speed and query performance. By offloading aggregation computations to the database and simplifying mapping code, this approach reduces data processing overhead and accelerates queries, especially when dealing with large volumes of time-based data.
+
+## Why Timeseries and Aggregations Are a Best Practice
+
+1. Improved Indexing Time • Less Data to Load: Mappings handle less data since raw data points are stored as immutable timeseries entities. • Database-Managed Aggregations: Aggregations are automatically computed by the database, reducing the workload on the mappings.
```

Comment:
Spacing issue that's distorting the formatting. I think the bullet point are suppose to be below the numbers, right?